src/test/java/com/google/sps/data/DatastoreEntityToObjectConverterTest.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.DatastoreEntityToObjectConverter;
+import com.google.sps.data.DatastoreManager;
+import com.google.sps.data.Post;
+import com.google.sps.data.Representative;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class DatastoreEntityToObjectConverterTest {
+    private final LocalServiceTestHelper helper =
+        new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
```

Comment:
please do this new LocalServiceTestHelper(...) in setup. We generally initialize instance variable in constructors. Since setUp is the Junit equivalent of a constructor, we do it there